ee/clickhouse/queries/trends/trend_event_query.py

```diff
@@ -83,7 +84,7 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
         return query, self.params
 
     def _determine_should_join_distinct_ids(self) -> None:
-        if self._entity.math == "dau":
+        if self._entity.math == "dau" and not self._aggregate_users_by_distinct_id:
```

Comment:
Not sure if we should do this for trends? Since it basically implies "Total Count" ?

Reply:
See [this test](https://github.com/PostHog/posthog/pull/8935/files/b7976138b360878a97b0e8791fe2cfae0c505980#diff-01a1ec955bb31f186af94c7b95360f1830997d2e31990c8eb3a191adaa0f6189R2347), so it's total count aggregated by distinct id, which for this customer would be exactly the same.

There's a test for person properties filtering in there as well. Obviously at that point there wouldn't be any speed improvements anymore, but again for this customer most of their queries don't have person property filtering.

Reply:
Gotcha, there's no aggregation in total count duh 🤦